arlas-server-xpack/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.arlas</groupId>
+    <artifactId>arlas-server-xpack</artifactId>
+    <version>11.7.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>ARLAS server for ES cloud</name>
```

Comment:
It is not for the cloud, it is for the x-pack

Reply:
fixed